node-services/sharing-service/controllers/sharedWithOthers.js

```diff
@@ -0,0 +1,63 @@
+const db = require('../dbInstance');
+const { validateEmail, validateUserParameter } = require('./validateParameters');
+
+// checkIfExists responseCodes
+// 0 -> Report not shared beforehand
```

Comment:
These response codes will get difficult to manage with time, any chance this can be a shared enum (or its equivalent in JS), preferably with meaningful string ,which both this and caller of these function can share. If not at least can these ResponseCodes be string?